lib/screens/profile_screens/profile/components/switch_account_bottom_sheet/interactor/viewmodels/switch_account_bloc.dart

```diff
@@ -35,7 +34,7 @@ class SwitchAccountBloc extends Bloc<SwitchAccountEvent, SwitchAccountState> {
     if (publicKeys.contains(null) || publicKeys.contains('')) {
       emit(state.copyWith(pageState: PageState.failure, error: ImportKeyError.InvalidPrivateKey));
     } else {
-      final results = await ImportAccountsUseCase().run(publicKeys as List<String>);
+      final results = await ImportAccountsUseCase().run(List<String>.from(publicKeys));
```

Comment:
I don't know why but on iOS it threw me the wrong cast exception, which didn't happen on Android. We must test always in both platforms.